### ISSUE TITLE
ESLint requires trailing comma for function parameters.

### ISSUE THREAD

**Issue by HouCoder** [eslint/eslint#8700]
**Tell us about your environment**

* **ESLint Version:** v3.19.0
* **Node Version:** v6.10.0
* **npm Version:** 5.0.3

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

```
{
    "extends": "eslint-config-airbnb-base",
    "parserOptions": {
        "ecmaVersion": 6,
        "sourceType": "module",
    },
    "globals": {
        "ga": true,
    },
    "env": {
        "browser": true,
        "node": true,
        "es6": true,
        "jquery": true,
    },
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
function extract(a, b, c) {
  console.log(c);
}

extract(
    'style-loader',
  [
    'css?sourceMap',
    'postcss',
    'sass?sourceMap',
  ],
  {
    publicPath: '../',
  } //👈 ESLint error: Missing trailing comma on the last function parameter.
);
```

**What did you expect to happen?**

ESLint should not check trailing comma for function parameter.

**What actually happened? Please include the actual, raw output from ESLint.**

➜  demo git:(dev) ✗ eslint eslint-test.js

/Users/tonni/Projects/demo/eslint-test.js
   2:3  warning  Unexpected console statement  no-console
  14:4  error    Missing trailing comma        comma-dangle

✖ 2 problems (1 error, 1 warning)

**Comment by not-an-aardvark** [comment]
Thanks for the report, but this is working as intended -- you are extending the `airbnb-base` configuration, which requires trailing commas for function arguments.

Also see: https://github.com/eslint/eslint/issues/8513, https://github.com/eslint/eslint/issues/7851, https://github.com/eslint/eslint/issues/7749, https://github.com/eslint/eslint/issues/7571.

**Comment by HouCoder** [comment]
👍 Thanks for your excellent explanation, closing it. @not-an-aardvark
